frameworks/PHP/symfony/src/Controller/DbController.php

```diff
@@ -58,18 +57,27 @@ public function update(Request $request): JsonResponse
         $queries = min(500, max(1, $queries));
 
         $worlds = [];
-
-        for ($i = 0; $i < $queries; ++$i) {
-            $world = $this->worldRepository->find(mt_rand(1, 10000));
-            if ($world) {
-                $randomNumber = mt_rand(1, 10000);
-                $world->setRandomNumber($randomNumber);
-                $worlds[] = $world;
-            }
+        $ids = $this->getUniqueRandomNumbers($queries);
+        sort($ids);
```

Comment:
The difference is marginal, it's more for harmonization, but I used the `ksort` in the `getUniqueRandomNumbers` function.
https://github.com/TechEmpower/FrameworkBenchmarks/blob/22998dd14c1badfe591f48156abc75ce96483001/frameworks/PHP/symfony/src/Controller/DbController.php#L26

Reply:
the method is used in `/queries` without need to be ordered.